core/src/main/java/io/jenkins/pluginhealth/scoring/probes/DependabotProbe.java

```diff
@@ -24,40 +24,17 @@
 
 package io.jenkins.pluginhealth.scoring.probes;
 
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.util.stream.Stream;
-
-import io.jenkins.pluginhealth.scoring.model.Plugin;
-import io.jenkins.pluginhealth.scoring.model.ProbeResult;
-
 import org.springframework.core.annotation.Order;
 import org.springframework.stereotype.Component;
 
 @Component
 @Order(DependabotProbe.ORDER)
-public class DependabotProbe extends Probe {
+public class DependabotProbe extends AbstractDetectBotConfigurationProbe {
     public static final int ORDER = LastCommitDateProbe.ORDER + 100;
```

Comment:
The Renovate and Dependabot probes are dependant only on `LastCommitProbe` which is already added in the `ORDER` of the abstract class.  That is why I did not add `+10` and `+20` where they are extended.


Am I missing something here?

Reply:
The order doesn't have to be strict because I fear that at some point we need a map of the probe and their order to be able to navigate the different values. So, when there is a strict requirement, we do the `+100`, when there is a strict requirement and a relation between the probe (like dependabot and dependabotpullrequest) we do the `+10`. This needs to be documented, that's on me.